fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java

```diff
@@ -153,19 +216,43 @@ private void relocateAndBalanceGroup() {
                 continue;
             }
 
+            // Won't make new relocation decision if there is still colocate relocation tasks in schedule.
```

Comment:
What's the purpose of serial execution?

Reply:
Like the comments said, we don't want to complicate the relocation logic and concurrent relocation decision won't accelerate the scheduling process. This is also to make repair task precede over balance task.